### DISCUSSION TITLE
Tilesheet mode [feature proposal]

### DISCUSSION THREAD

**Discussion by lapspider45** [Orama-Interactive/Pixelorama#597] [Ideas]
The current "Tile mode" is good for editing single tiles, but when making art that uses many tiles (such as a game), you often want to edit multiple tiles at once. 
I would like a tool or a mode that lets you place the tiles in a tilemap to see how they look together.

### Features:
* assigning rects from the image to tiles (similar to the current Godot tileset editor)
* painting tiles onto a grid in a separate view (like the tilemap in Godot)
* the tilemap view updates when you edit the image to reflect the modified tiles

### Extras that would be nice to have in the future:
* save the tilemap view to png
* exporting the tiles to a Godot tileset `.tres` (Godot 4 tilemap overhaul will break the current format?)
* exporting the tilemap to json, tmx, .tscn, what have you
* editing the tiles through the tilemap view
* "focusing" on one tile, highlighting that tile in the tilemap and letting you edit only that tile

**Comment by jordanlis** [comment]
Hello, just would like to know where is this proposal ? This is a must have feature I think. 
According to me, this is one of the main reason why people use aseprite instead of pixelorama.

**Reply by OverloadedOrama** [reply]
It will eventually happen, but not before v1.0 is released. Also, this proposal seems to be the same as #611, right? Unless I'm missing something.

**Comment by jordanlis** [comment]
Yep, exactly the same proposal. OK so is there any date for v 1? Just to know, no rush people will continue to work on other tool unti then

**Reply by OverloadedOrama** [reply]
No date yet, but I estimate that because v1.0 will see the project being ported from Godot 3.5 to Godot 4.x, the process will take quite a lot of months, so I wouldn't expect it before 2024. The proposal will likely be implemented at some 1.x release, unless a contributor wants to do it earlier.
